src/wh_11_1/Test.java

```diff
@@ -0,0 +1,109 @@
+package wh_11_1;
+
+import java.io.*;
+import java.nio.file.*;
+
+class Test {
+    public static void main(String[] args) {
+        final String FILE_NAME = "TestFile.txt";
+        final String RESULTS_FILE_NAME = "testResults.txt";
+        try {
+            Employe[] employes = EmoloyeeReader.readFromFile(FILE_NAME);
+            for (Employe x : employes) {
```

Comment:
'x' to zla nazwa. lepiej uzywac pelnej nazwy  `employee`

Reply:
zmienilem :)